src/website/views/components/tooltip/_examples.pug

```diff
@@ -24,6 +24,30 @@ p.-text
 
       <script>chi.tooltip(document.getElementById('tooltip-1'));</script>
 
+h3 Light
+p.-text
+  | To render a light color tooltip, apply the <code>data-tooltip-color="light"</code> attribute to Tooltip trigger element.
```

Comment:
Is this the rules you're following for style now?
I would suggest using custom attributes for Custom Elements only and keep on using classes for CSS only solutions.

Reply:
In this case, we thought it would be wrong to set the additional class on Tooltip trigger button since it would be confused with the styling of the button itself

Reply:
Hmmm, understand, and agree.

Reply:
Thanks Dani!